src/modules/universal-discovery/components/select-content-button/css/select.content.button.component.css

```diff
@@ -0,0 +1,14 @@
+.c-select-content-button {
+	position: absolute;
+	right: 1.3rem;
+	top: 50%;
```

Comment:
It shouldn't be here. Because it's an information about block position. It should go to parent container style definitions.